src/vm/dwreport.cpp

```diff
@@ -2426,9 +2440,11 @@ FaultReportResult DoFaultReportWorker(      // Was Watson attempted, successful?
     pWatsonSharedMemory->bfmsoctdsLetRun = offerFlags;
 
     {
+        PathString wzModuleFileName;
         DWORD dwRet = WszGetModuleFileName(NULL,
-	                                       pWatsonSharedMemory->wzModuleFileName,
-	                                       NumItems(pWatsonSharedMemory->wzModuleFileName));
+	                                       wzModuleFileName);
+        BaseBucketParamsManager::CopyStringToBucket(pWatsonSharedMemory->wzModuleFileName, NumItems(pWatsonSharedMemory->wzModuleFileName), wzModuleFileName);
```

Comment:
The old code did not have to care about truncation as paths greater than MAX_PATH were not ever used. This PR is to enable it. If we blindly truncate it we are likely to get only the leading characters of the module path and probably never get the base name of the module- Is this what you want as the behaviour. Also @noahfalk  was saying this will not be hit in Win7 and above- which is the current focus of  this feature.